plugins/ImageGraph/StaticGraph.php

```diff
@@ -248,14 +265,17 @@ protected function initpData()
             }
         }
 
-        // Use a different formating method if not using unifont
+        // Fix whitespace if not using unifont
+        $abscissaSeries = $this->abscissaSeries;
         $formatMethodName = 'formatYAxis';
-        if (strpos($this->font, 'unifont') === false) {
+
+        if (!str_ends_with($this->font, API::UNICODE_FONT)) {
```

Comment:
I guess we actually should be able to, as we have the symfony php8 polyfill included.

Reply:
I change it back to `false === strpos`, but as said the polyfill is already imported by symfony and twig.

I have no strong opinion on using the polyfill functions, but I would not expect them to silently go away and break things.

Reply:
> I guess we actually should be able to, as we have the symfony php8 polyfill included.

I probably missed that. It might be good if we had some docs on which PHP8 functions we could use so that we actually write better code.

Ref #21736.

Reply:
Based on my quick search we don't have a single use of `str_ends_with` or `str_starts_with` in the codebase, so I would probably say we should not introduce it randomly here. Thoughts?

Reply:
I removed the usage. We could add the already existing polyfills explicitly to our `composer.json` to announce their usability. Perhaps both the functions you mentioned and also `str_contains` then get some usage, eventually replace the several hundred `strpos` occurences in the codebase :see_no_evil:

Reply:
sounds like a good approach @mneudert. Not sure if it's worth to create a separate issue for that or wait until someone has some time in between to handle that

Reply:
Sounds good as a separate technical debt issue, thanks Marc!